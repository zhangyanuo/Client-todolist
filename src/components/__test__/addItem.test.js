import AddItems from '../AddItems';
import React from 'react';
import {shallow} from 'enzyme';

test('render AddItems component',()=>{
    const wrapper=shallow(
        <AddItems/>
    )
    expect(wrapper).toMatchSnapshot();
})
//when given then //方法 ，交互，长得什么样子--->某个元素中是否包含你想要的东西 ,事件//react-test-library