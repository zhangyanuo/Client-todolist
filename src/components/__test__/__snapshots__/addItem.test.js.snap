// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItems render the text inside it 1`] = `ShallowWrapper {}`;

exports[`render AddItems component 1`] = `ShallowWrapper {}`;
